web/cypress/integration/hfi-calculator-page.spec.ts

```diff
@@ -2,10 +2,23 @@ import { HFI_CALC_ROUTE } from '../../src/utils/constants'
 
 describe('HFI Calculator Page', () => {
   beforeEach(() => {
+    cy.intercept('GET', 'api/hfi-calc/', { fixture: 'hfi-calc/stations.json' }).as('getFireCentres')
```

Comment:
Should this be: `hfi-calc/`, like `morecast`, `percentile-calculator` and `c-haines`?

Reply:
it's intercepting the call to the API to get the wx stations relevant to the HFI calculator. Morecast & Percentile Calculator do the same thing as this.